src/kvstore/RocksEngine.h

```diff
@@ -70,6 +85,11 @@ class RocksPrefixIter : public KVIterator {
   RocksPrefixIter(std::unique_ptr<rocksdb::Iterator> iter, rocksdb::Slice prefix)
       : iter_(std::move(iter)), prefix_(prefix) {}
 
+  RocksPrefixIter(rocksdb::Slice prefix) : prefix_(prefix) {
```

Comment:
`Single-parameter constructors should be marked explicit.  [runtime/explicit] [5]`

[src/kvstore/RocksEngine.h:88](https://github.com/vesoft-inc/nebula/actions/runs/4778193665/jobs/8494456333?pr=5525#step:6:9)